libs/@guardian/eslint-plugin-source-foundations/package.json

```diff
@@ -38,8 +38,8 @@
 		"wireit": "0.14.4"
 	},
 	"peerDependencies": {
-		"@guardian/libs": "^16.0.0",
-		"@guardian/source-foundations": "^14.1.4",
+		"@guardian/libs": "workspace:^",
+		"@guardian/source-foundations": "workspace:^",
```

Comment:
no it would be the latter (see https://pnpm.io/workspaces#publishing-workspace-packages), which makes it tricky.

it means needing to publish a new version of package with the peerdep whenever the workspace version of the peerdep changes, even if that change is not necessary for the package with the peerdep

Reply:
Ah yeah you’re right the examples does not turn `workspace:^` into `^1.0.0`!

Reply:
Yea, This uses the version that is currently in the repo for both `workspace:*/^`. The benefit I was thinking was that it would mean we could surface issues in new versions by consuming them as they are developed. By directly consuming the built version locally. But I think it comes with some additional complications and excessive bumping of packages

Reply:
It would unfortunately need to use `^16.5.4` as you cannot be sure that it would work with an earlier version if you are developing and testing against that version.